test/e2e/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit  # Exit with error on non-zero exit codes
+set -o pipefail # Check the exit code of *all* commands in a pipeline
+set -o nounset  # Error if accessing an unbound variable
+set -x          # Print each command as it is run
+
+docker build . \
+       --build-arg SECRET_STORE_VERSION=v0.0.12 \
```

Comment:
I think this should be `${SECRET_STORE_VERSION}`

Reply:
Yup